### DISCUSSION TITLE
有做表格的计划嘛

### DISCUSSION THREAD

**Discussion by hzbnb** [sutras/sard-uniapp#58] [Ideas]
类型这样
![image](https://github.com/user-attachments/assets/6447aa06-8109-4af1-9c09-5be14329f3c6)

**Comment by sutras** [comment]
在此组件库开发阶段前的设计阶段，已将表格组件列在了规划中，计划开发的功能如下：
1. 表头
2. 单元格
3. 边框
4. 固定表头
5. 固定列
6. 斑马线
7. 多级表头
8. 合并单元格

但因小程序端的表格组件使用场景比较少，而简单的表格又可以通过flex布局实现，且合并单元格的复杂度又较高，故而一直推延。
既然有需求，那就尝试去实现这个表格组件。

**Reply by hzbnb** [reply]
好滴 谢谢大佬 表格在做一些手机端的展示数据或者管理的时候偶尔要用到。
可以做一个简单一点的表格 功能可以不想桌面那么强大。

**Comment by sutras** [comment]
v1.9.0新增了表格组件

**Comment by elarbor** [comment]
sard的样式总是很优雅啊

**Reply by hzbnb** [reply]
是的 是的
